pages/common/xh.md

```diff
@@ -0,0 +1,28 @@
+# xh
+
+> XH: Friendly and fast tool for sending HTTP requests.
+> More information: <https://github.com/ducaale/xh>.
+
+- Send a GET request:
+
+`xh {{httpbin.org/get}}`
+
+- Send a POST request with JSON body: `{"name": "ahmed", "age": 25}`:
+
+`xh post {{httpbin.org/post}} {{name=john}} {{age:=25}}`
+
+- Send a GET request with querystring id=5&sort=true:
+
+`xh get {{httpbin.org/get}} {{id==5}} {{sort==true}}`
+
+- Send a GET request and include a header named x-api-key with value 12345:
+
+`xh get {{httpbin.org/get}} {{x-api-key:12345}}`
+
+- Send a PUT request and pipe the result to less:
+
+`xh put {{httpbin.org/put}} {{id:=49}} {{age:=25}} | less`
```

Comment:
This example doesn't seem particularly useful—anyone familiar with the command line will understand that the stdout of one command may be passed to the stdin of another; `less` is frequently to demonstrate this in tutorials anyway, hence it doesn't strike me as being useful to include.

Any reason for including it?

Reply:
Hmm, okay, I'll remove that.